### PR TITLE
Fixes #106 - Docker quickstart

### LINKED ISSUE #106: As developer, I want a quickstart docker-compose and procedure to get started quickly
- [x] Docker compose like: https://github.com/maykinmedia/objects-api/blob/master/docker-compose-quickstart.yml
- [x] Something like this: https://objects-and-objecttypes-api.readthedocs.io/en/latest/installation/quickstart.html
- [x] Something like this in the README: https://github.com/maykinmedia/objects-api#quickstart

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #106 

A docker-compose-quickstart.yml was added along with instructions on how to use it.

### CODE REVIEW

File: docker-compose-quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - ./docker-init-db.sql:/docker-entrypoint-initdb.d/init_db.sql
+
+  cache:
+    image: redis
+
+  web:
+    build: .
```
Comment (annashamray) [inline docker-compose-quickstart.yml:18]: Hmm, I think quick-start means that we will use the existing image without building it. @joeribekker is it what you wanted?

**Review by annashamray — CHANGES_REQUESTED** [review]
@SilviaAmAm The documentation is very nice! 

@joeribekker are fixtures required for the quick start? You can't use the rma app without service config, but I don't think that putting credentials to openzaak into fixture is a good idea

File: docker-compose-quickstart.yml (hunk 1 above)
Comment (joeribekker) [reply docker-compose-quickstart.yml:18]: Correct, default docker-compose is probably `build: .` and the quickstart has a pre-build image, like `image: record-management-app:latest` or so.

File: README.rst [hunk 2]
```diff
@@ -29,6 +29,22 @@ system actions.
...
 The RMA supports all backends implementing the 1.0.x `API's voor Zaakgericht Werken`_.
 
+Quickstart
+==========
+
+A `docker-compose-quickstart.yml`_ is provided to get up and running quickly. To run the container:
+
+    .. code:: shell
+
+        $ git clone https://github.com/maykinmedia/record-management-app.git
```
Comment (annashamray) [inline README.rst:39]: Now when you use the image from docker hub you don't need to clone the repo
You can just download `docker-compose-quickstart.yml` with `wget` or `curl`

File: docker-compose-quickstart.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      - DEBUG=True
+      - CACHE_DEFAULT=cache:6379/0
+      - CACHE_AXES=cache:6379/0
+      - CACHE_OAS=cache:6379/1
+      - SECRET_KEY=${SECRET_KEY:-145cqii@)+z$b&9dcmdctpm+&be1#9zslxm+x)0+u*^qf!qz(6}
+    ports:
+      - 8000:8000
+    depends_on:
+      - db
+      - cache
```
Comment (annashamray) [inline docker-compose-quickstart.yml:31]: Sorry, but one more thing, which I've just noticed - all docker-compose files lacks celery containers, which are needed to perform the selected zaken removal.
You can look at BPTL docker-compose file for some inspiration.

File: docker-compose-quickstart.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    image: postgres
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+
+  redis:
+    image: redis
+
+  web:
+    image: maykinmedia/record-management-app:latest
+    image: maykinmedia/record-management-app:latest
```
Comment (annashamray) [inline docker-compose-quickstart.yml:17]: duplicated line

File: docker-compose-quickstart.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  web:
+    image: maykinmedia/record-management-app:latest
+    environment: &web_env
+      - DJANGO_SETTINGS_MODULE=rma.conf.docker
+      - ALLOWED_HOSTS=*
+      - DEBUG=True
+      - CACHE_DEFAULT=redis:6379/0
+      - CACHE_AXES=redis:6379/0
+      - CACHE_OAS=redis:6379/1
+      - SECRET_KEY=${SECRET_KEY:-145cqii@)+z$b&9dcmdctpm+&be1#9zslxm+x)0+u*^qf!qz(6}
```
Comment (annashamray) [inline docker-compose-quickstart.yml:24]: CELERY_* environment variables are missing

**Review by annashamray — APPROVED** [review]

**Pull request merged**